addons/l10n_fi/__manifest__.py

```diff
@@ -0,0 +1,38 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    ODOO Addon module by Sprintit Ltd
+#    Copyright (C) 2018 Sprintit Ltd (<http://sprintit.fi>).
+#
+#    Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
I don't know if we should keep these #########, I don't really see it elsewhere

Reply:
I copied the header from Belgium. Should I keep the copyright notice ?